blockchain/citizens/become-congressmen.md

```diff
@@ -0,0 +1,142 @@
+
+#### How to apply
+
+1. Visit [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmainnet.liberland.org#/council)
+2. Click "Submit candidacy"
+3. Select the account you want to apply with then click "Sign and Submit"
+
+<br />
+<img height="200px" src="congress-assets/submit-council-candidacy.png"/>
+<br />
+
+After applying, other citizens can vote for you and other candidates.
+
+### Congressmen's actions
+
+Congressmen can submit proposals under the ["Motions" tab](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmainnet.liberland.org#/council/motions). Then these can be considered by the whole Congress.
+
+If you do not know what proposals you can make, find out more at "How to submit proposals"
+
+#### How to vote for a proposal
+
+All pending proposals are displayed in the ["Motions" tab](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmainnet.liberland.org#/council/motions). Proposals are displayed as a list and any congressman can vote on them.
+
+As you can see, next to the motion there is a vote button. It opens the voting panel. Then on the panel, you can vote for or against the proposal
+
+<br />
+<img height="300px" src="congress-assets/motions-view.png"/>
+<br />
+
+#### How to submit proposals
+
+To vote, you need to set the correct threshold and proposal. Different actions may require different minimum thresholds. Threshold is the minimum percentage of votes that a proposal must receive in order to be approved. The threshold is automatically set to 50% but some methods may require setting a different threshold value. To find out which proposals should have what value, go to the "Threshold"
+
+1. Open the ["Motions" tab](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmainnet.liberland.org#/council/motions)
+2. Click the "Propose motion" button
+3. Fill in the information about the proposal and then click the "Propose" button
+
+<br />
+<img height="300px" src="congress-assets/add-proposal.png"/>
+<br />
+
+### Liberland legislation
+
+Congressmen are able to add and remove legislation at the international treaty tier. The international treaty tier is represented as "tier=1". Learn more about these methods in `Thresholds`.
+
+In Liberland, there are three tiers of international treaties: the "Constitution," "International Treaty," and "Law." These legal frameworks can only be altered by specific groups of individuals.
+
+- Constitution: This tier encompasses the rights outlined in the constitution. Any citizen can propose changes to this law.
+- International Treaties: These treaties fall under the jurisdiction of the congress. Modifications to these treaties can be made through "motions" initiated by the congress.
+- Law: This tier comprises legislation. Any citizen has the opportunity to submit their own proposals.
+
+### Democracy external
+
+Congress can create external proposals that differ from the usual proposal only in priority. Below are steps presenting the addition of such a proposal.
```

Comment:
what does 'differ in priority' mean ?

They differ in the ratio of yes/no votes on the referendum required for it to pass, if i recall correctly. Please expand on the differences in the referendums created